content/spire/try/oidc_federation_aws.md

```diff
@@ -0,0 +1,399 @@
+
+* [SPIRE Server and Agent](/downloads/) executables
+* Ability to create instances to contain the SPIRE Server and Agent
+* Root access to the instances that contain the SPIRE Server and Agent
+* Network access to an ACME-compliant CA such as Let's Encrypt
+* Access to the AWS console
+* [AWS CLI tools](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) installed on the same instance as the SPIRE Agent
+* DNS A record and TCP/IP access to port 443 for the SPIRE OIDC [Discovery Provider REST API](/spire/docs/oidc_discovery/)
+
+# Part One: Configure SPIRE Components
+
+In the first half of this procedure, you configure the SPIRE components. In the second half, you configure the AWS components.
+
+## Configuration Overview for SPIRE Components
+
+The table below summarizes the steps needed to configure the SPIRE components of OIDC federation to AWS. Follow the links for details.
+
+| Step | Action | Description |
+| --- | --- | --- |
+| Step 1 | [Prepare installation environments](#prepare-installation-environments) | Decide where you'll install SPIRE components |
+| Step 2 | [Install, configure, and run the SPIRE Server](#install-configure-and-run-the-spire-server) | Customize `server.conf`, including `ca_key_type` and `jwt_issuer` settings |
+| Step 3 | [Install, configure, and run the OIDC Discovery Provider](#install-configure-and-run-the-oidc-discovery-provider) | The SPIRE OIDC Discovery Provider serves OIDC discovery documents via a REST API |
+| Step 4 | [Install and configure SPIRE Agent](#install-and-configure-the-spire-agent) | Configure the SPIRE Agent as needed for your environment |
+| Step 5 | [Register (Attest) the Node](#register-attest-the-node) | Verify the identity of the node (machine) that the SPIRE Agent runs on |
+| Step 6 | [Register Workloads on the SPIRE Server](#register-workloads-on-the-spire-server) | Verify the identity of workloads |
+
+# Configure SPIRE for OIDC Federation
+
+Follow the procedures in the sections below to configure SPIRE for OIDC federation.
+
+## Prepare Installation Environments
+
+In a typical deployment, you would configure one machine to run the SPIRE Server and OIDC Discovery Provider and a second machine to run the SPIRE Agent. The machines can be a physical machine, virtual machine, instance, or container. The examples in these instructions do not describe how to use Kubernetes or Docker commands to set up OIDC federation, but they are supported.
+
+Review the [requirements](#required-accounts-and-permissions) described above.
+
+## Install, Configure, and Run the SPIRE Server
+
+The steps below provide an overview of installing and configuring the SPIRE Server. For more detailed information, see [Install SPIRE Server](/spire/docs/install-server/).
+
+1. Set up an instance or machine on which to run the SPIRE Server and OIDC Discovery Provider.
+
+2. [Download](/downloads/) and [install](/spire/docs/install-server/) SPIRE Server.
```

Comment:
This instruction is pretty open-ended, and there's lots of ways the user might fall off the rails trying to follow it. Instructions in a tutorial need to be specific.

To save time, you might want to build a tutorial that builds on either the Linux or Kubernetes quickstart, and picks up where that leaves off. I'd strongly recommend the Kubernetes QS since about 2x folks seem to be using that right now over the regular Linux one.